src/lib/utils/number.ts

```diff
@@ -12,3 +12,17 @@ export const formatEthers = (number: string) => {
 	const asNumber = Number(ethers.utils.formatEther(number));
 	return formatNumber(asNumber);
 };
+
+export const ONE_DECIMAL_PLACE_VALUE = 1;
+export const TWO_DECIMAL_PLACE_VALUE = 2;
+export const THREE_DECIMAL_PLACE_VALUE = 3;
+
+export const formatByDecimalPlace = (
+	value: number | string,
+	places: number,
```

Comment:
If we want to restrict as mentioned above then the type needs to be modified from number

Reply:
as comment above